README.md

```diff
@@ -101,51 +101,45 @@ Please see [this file](./__tests__/README.md) for more info.
 ###### Mentor intake schema:
 
```

Comment:
This change is a reflection of pulling from main when other pr's had been merged so this file matches the most recent main branch at the time.